src/js/modules/setup/controllers/change-password/change-password.ctrl.js

```diff
@@ -0,0 +1,305 @@
+
+    angular.module("blocktrail.setup")
+        .controller("SetupChangePasswordCtrl", SetupChangePasswordCtrl);
+
+    function SetupChangePasswordCtrl($scope, $stateParams, $http, $q, $sce, PasswordStrength, dialogService, $log, $filter, sdkService, $translate, CONFIG) {
+        $scope.working  = false;
+        $scope.error    = null;
+        $scope.form     = {
```

Comment:
NIT: lets stop doing this aligning of `=` signs ;)  
not gonna fix it retroactively but we shouldn't continue this boaz-practice :P